eth/handler.go

```diff
@@ -189,7 +189,7 @@ func (pm *ProtocolManager) removePeer(id string) {
 	if peer == nil {
 		return
 	}
-	log.Debug("Removing Ethereum peer", "peer", id)
+	log.Proto("Removing Ethereum peer", "peer", peer.ID())
```

Comment:
@SimonSK - this is the only place I monitor removal of Ethereum peers, is this sufficient? Will I be able to keep track of the number of peers by just tracking the "peer connected" and this message?